main_test.go

```diff
@@ -49,3 +49,7 @@ func TestMain(m *testing.M) {
 
 	os.Exit(ret)
 }
+
+func createEmptyFile(path string) (err error) {
```

Comment:
Generic files like "common.go", "lib.go", "utils.go" always end up with a mixed bag of "things". Could maaaybe be fileutils_test.go, it's more definite already.